src/service/filtering/generic_view.py

```diff
@@ -0,0 +1,21 @@
+from src.common.product_models.columnar_attribs_common_models import ColumnarAttributesSpec, AttributesColumnSpec
+
+_FIELD_COL = {
+    "kbase_display_name": {
+        "type": "string",
+        "filter_strategy": "ngram",
+    }
+}
+
+
+def create_generic_spec() -> ColumnarAttributesSpec:
+    """
+    Create a generic spec for a collection.
+
+    Arango search includeAllFields will be set to true. This means that all fields will be indexed
+    by default except for those that are explicitly defined in the _FIELD_COL dict.
```

Comment:
```suggestion
    This spec only includes fields common to most or all collections that require special handling and it is
    expected that all other fields are handled in a generic way.
```

Edited to avoid mentioning internals in external docs and remove references to Arango, which this code shouldn't know about.